src/Kris/LaravelFormBuilder/Events/BeforeFormValidation.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace Kris\LaravelFormBuilder\Events;
```

Comment:
I don't know when validation happens. When the caller calls `isValid()` ? What if it's called multiple times? BeforeValidation is useful because that's the time to add logic to pass or fail the validation with custom logic. AfterValidation can't really do anything useful...